dutch/bos/donker-bos.md

```diff
@@ -1,8 +1,10 @@
 In het donkere bos aangekomen, hoor je plotseling stemmen.
 Je verstaat geen woord ervan, maar de stemmen komen wel steeds sneller naar je toe.
 
-Je verstopt je in de struiken, en ziet twee mannen met een lijkzak sleuren.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.